### ISSUE TITLE
linuxkit run vmware -data and -data-file support

### ISSUE THREAD

**Issue by scriptjs** [linuxkit/linuxkit#3129]
**Description**
linuxkit vmware run lacks support for `-data` and `-data-file` options. This is likely already know but was unable to find an issue for tracking.  Was able to find this issue but this did not specifically deal with the lack of support.

https://github.com/linuxkit/linuxkit/issues/1732

**Steps to reproduce the issue:**
`linuxkit -v run vmware -mem 2048 -data-file user-data.json linuxkit`

**Describe the results you received:**
```
flag provided but not defined: -data-file
```
**Describe the results you expected:**
An iso will be created and mounted with access to the data on boot.

**Comment by justincormack** [comment]
Yes, it would be nice to add these; I added support to a few more platforms recently. It should not be too hard to add if you just want to add an ISO.

**Comment by scriptjs** [comment]
I will help with some of these things. In the meantime, I just want the issues identified so they can be tracked. I have a couple of other issues to file on the basis of some work I've been doing, then expect to contribute some code to the project.
